test/recipes/80-test_cmp_http_data/test_credentials.csv

```diff
@@ -44,3 +44,6 @@ expected,description, -section,val, -ref,val, -secret,val, -cert,val, -key,val,
 0,multiple digests, -section,,BLANK,,BLANK,, -cert,signer.crt, -key,signer.p12, -keypass,pass:12345,BLANK,,BLANK,, -digest,sha256 sha512,BLANK,
 ,,,,,,,,,,,,,,,,,,,,,,
 0,unprotected request, -section,,BLANK,,BLANK,, -cert,"""", -key,"""", -keypass,"""",BLANK,,BLANK,,BLANK,, -unprotected_requests,
+,,,,,,,,,,,,,,,,,,,,,,
+1,client with KEM, -section,,,,,, -cert,kem.crt, -key,kem.key,,,BLANK,,BLANK,,BLANK,,BLANK,,,,
+1,server with kem, -section,,BLANK,,-secret,"""", -cert,signer.crt, -key,signer.p12, -keypass,pass:12345,BLANK,,BLANK,,BLANK,,BLANK,,-srvcert, server_kem.crt
```

Comment:
Using `-srvcert` this way gets silently ignored.
Unfortunately, we'll need to spin up a new server instance for making it use other (KEM) credentials.